README.md

```diff
@@ -1,11 +1,83 @@
-# Explorer Template
 
-This is a template to help you get started with your projects. Find out more information inside of the Fello Handbook.
+![pod-3.1.3-team-3](https://socialify.git.ci/MLH-Fellowship/pod-3.1.3-team-3/image?descriptionEditable=Maintain%20peak%20productivity%20by%20knowing%20exactly%20what%20you%20need%20to%20work%20on%20when%20you%20need%20to%20work%20on%20it.&font=Rokkitt&issues=1&language=1&owner=1&pattern=Brick%20Wall&pulls=1&stargazers=1&theme=Light)
+
+[![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/made-with-javascript.svg)](https://forthebadge.com)
+# GitDash 
+
+> Maintain peak productivity by knowing exactly what you need to work on when you need to work on it. 
+
+## Introduction
+
+As software developers, we have often started new projects and then we forgot about it. This mostly happens because we don't track what items we are working on and what the deadline is. We might track a single repo and all the issues on it but what if we had one place which would track all of our issues and send us reminders when we don't work a specific deadline irrespective of what the repository may be. Presenting to you, GitDash...
+
+![homepage](images/homepage.PNG?raw=True)
+
+## Documentation and other details
+
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First install the dependencies for the project:
+
+```bash
+cd gitdash
+npm install
+```
+
+Then, run the development server:
+
+```bash
+npm run dev
+```
```

Comment:
This is some great markdown. Love the inclusion of `bash` to get some syntax highlighting in there!